doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| DEVICE_NAME | String | Yes | The server name  |
+<p align="center"><b>Table 6-3-12:</b> SDNC Nodes Information.</p>
+
+<a name="6.3.13"></a>
+### 6.3.13 Storage cluster information 
+Definition of storage cluster and storage pool, 
+
+#### 6.3.13.1 Storage pool plan
+Storage pool name in each storage cluster, and nodes in Storage pool should be defined, so the storage installer will know which nodes are installing.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| STORAGE_CLUSTER_NAME | String | Yes |   |
+| STORAGE_POOL_NAME | String | Yes |  |
+| DEVICE_NAME | String | Yes |  |
+
+<p align="center"><b>Table 6-3-13-1:</b> Storage Pool Plan </p>
+
+#### 6.3.13.2 Distribution storage pool info
+Storage pool information, defines the management account and network information
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| STORAGE_CLUSTER_NAME | String | Yes |   |
+| NODE_POOL | String | Yes |  |
+| DISK_POOL_NAME | String | Yes |  |
+| STORAGE_POOL_NAME | String | Yes |  |
+| HA_NAME_LIST | String | Yes |  |
+| AZ_NAME | List | Yes |  |
+| STORAGE_POOL_NODE_ACCOUNT | List | Yes |  |
+| MAX_QUOTA_CAPACITY | List | Yes |  |
+| STORAGE_POOL_MANAGEMENT_IP | List | Yes |   |
+| NETWORK_MASK | List | Yes |   |
+| GATEWAY | List | Yes |   |
+| VIM_USER | List | Yes |   |
+| VIM_PASSWORD | List | Yes |   |
+| PIM_USER | List | Yes |   |
+| PIM_PASSWORD | List | Yes |   |
+| STORAGE_CLUSTER_SERVICE_IP | List | Yes |   |
+| STORAGE_CLUSTER_SERVICE_GW | List | Yes |   |
+| STORAGE_CLUSTER_BACKEND_IP | List | Yes |   |
+| STORAGE_CLUSTER_BACKEND_GW | List | Yes |   |
+| BACKUP_POLICY | List | Yes |   |
+
+<p align="center"><b>Table 6-3-13-2:</b> Distribution storage pool info.</p>
+
+
+<a name="6.3.14"></a>
+### 6.3.14 Software integration information
+After VIM and Storage software installation finished, parameters willl be needed in integration process of VIM and Storage,
+the parameters should be defined in advance.
+
+#### 6.3.14.1 VIM Context
+Parameters from VIM vendor for integration.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes |  |
+| AUTHORIZATION | String | Yes | One-way or Two-way authentication |
+| JOINT_WAY | String | Yes | by ISCSI or client |
```

Comment:
Why iSCSI on the VIM context?

Reply:
It should be in the storage context, removed it